tests/drivers/spi/spi_loopback/boards/mimxrt1170_evk_cm7.conf

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright 2023 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+CONFIG_SPI_MCUX_LPSPI_DMA=y
```

Comment:
should make a testcase.yaml case for this and test other platforms than just 1170

Reply:
Please check this.

Reply:
see other comment